docs/proposals/version-documentation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Documentation Versioning"
+linkTitle: "Documentation Versioning"
+weight: 1
+slug: documentation-versioning
+---
+
+- Author: [Jay Batra](https://github.com/jaybatra26)
+- Date: March 2020
+- Status: proposal
+
+## Problem
+
+In Cortex, documentation needs a lot of reordering. The idea is to have version documentation just like Prometheus.[`Prometheus`](https://prometheus.io/docs/introduction/overview/). Documentation is the main source of information for current contributors and first-timers. A properly versioned documentation will help everyone to have a proper place to look for answers before flagging it in the community. 
+
+In this proposal, we want to solve this. In particular, we want to:
+
+1. Version Documentation.
+2. Add or remove necessary and unnecessary data from it.
+3. Organize the documentation so that it's easily understandable.
+
+## Proposed solution
+
+The idea is to use a tool to organize current documentation. Prometheus uses netlify. 
+Rucio uses Sphinx. Other tools like Github Pages
+
+### GitHub Pages
+
+#### Pros
+1. It has a very familiar interface.
+2. Easy to set up.
+3. Supports Jekyll out of the box.
+4. Supports custom domains. 
+
+#### Cons
+1. Currently, there is no support for HTTPS for custom domains. It’s probably coming soon though.
```

Comment:
HTTPS for custom domains works for some time now (I use it myself for at least a year)